lib/mayaUsd/ufe/PulledObjectHierarchy.h

```diff
@@ -58,7 +58,10 @@ class MAYAUSD_CORE_PUBLIC PulledObjectHierarchy : public Ufe::Hierarchy
     Ufe::SceneItem::Ptr sceneItem() const override;
     bool                hasChildren() const override;
     Ufe::SceneItemList  children() const override;
-    Ufe::SceneItemList  filteredChildren(const ChildFilter&) const override;
+#if defined(UFE_V4_FEATURES_AVAILABLE) && (UFE_MINOR_VERSION > 4 || UFE_PATCH_LEVEL > 3)
+    bool hasFilteredChildren(const ChildFilter&) const override;
+#endif
+    UFE_V2(Ufe::SceneItemList filteredChildren(const ChildFilter&) const override;)
```

Comment:
Why did you add the UFE_V2 here? It is not needed as this file is only included in Ufe v3